src/praisonai-bot/praisonai_bot/bots/_config_schema.py

```
@@ -16,6 +16,20 @@
+def _register_redaction(value: str) -> None:
+    """Register a resolved secret value for log redaction (best-effort).
+
+    Delegates to the core redaction registry (Issue #3102) but never fails
+    config validation if core is unavailable.
+    """
+    try:
+        from praisonaiagents.secrets import register_secret_for_redaction
+
+        register_secret_for_redaction(value)
+    except Exception:  # pragma: no cover - redaction is best-effort
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)